docs/Formatting.md

```diff
@@ -148,7 +148,8 @@ If this value is set to 0, then all blank lines will be removed during formattin
 ### xml.format.splitAttributes
 
   Set to `true` to split node attributes onto multiple lines during formatting. Defaults to `false`.
-  Overrides the behaviour of [xml.format.preserveAttributeLineBreaks](#xmlformatpreserveattributelinebreaks)
+  Overrides the behaviour of [xml.format.preserveAttributeLineBreaks](#xmlformatpreserveattributelinebreaks).
+  Please see [xml.format.numberOfIndentsForSplitAttributes](#xmlformatnumberOfIndentsForSplitAttributes) for information on configuring the indentation level of the attributes.
```

Comment:
splitAttributesIndentation is a tiny bit shorter

Reply:
HTML LS provides a similar settings with `html.format.wrapAttributesIndentSize`, to be consistent with HTML, I suggest `xml.format.spliAttributesIndentSize`.